src/components/Home/Home.jsx

```diff
@@ -188,10 +220,38 @@ class Home extends Component {
                         <Row
                             style={{
                                 padding: "10px"
+
                             }}
                         >
-                            <Button href="/personality">
+                            <Button href="/personality" style={{
+
+
+                                margin: "auto",
+                                left: "30%",
+                                bottom: "10px",
+                                position: "relative",
+
+                                alignItems: "center",
+                                justifyContent: "center",
+
+                                border: "2px solid #2D77A3",
+                                boxSizing: "borderBox",
+                                borderRadius: "30px",
+                                height: "40px",
+                                color: "#2D77A3",
+
+                                fontFamily: "Open Sans",
```

Comment:
Since it's repeating a lot, maybe we should move font-family and font-style to the body CSS